src/SFA.DAS.AdminService.Web/Views/Application/Sequence.cshtml

```diff
@@ -36,27 +36,51 @@
                             <dd class="govuk-summary-list__value">@Model.ApplicationReference</dd>
                         </div>
                     }
+                    @if (!string.IsNullOrWhiteSpace(Model.LegalName) && Model.IsWithdrawal)
+                    {
+                        <div class="govuk-summary-list__row">
+                            <dt class="govuk-summary-list__key">Organisation name</dt>
+                            <dd class="govuk-summary-list__value">@Model.LegalName</dd>
```

Comment:
This needs checking for correct name against the use on the tab view organisation name

Reply:
Changed to EPOA Assessor Name